tests/src/__tests__/test.js

```diff
@@ -115,7 +115,35 @@ for (let useInnerRef of [false, true]) {
     for (let testFun of funs) {
       test(testFun.name, async () => {
         await page.goto(testFile);
-        await page.evaluate(`render(${useInnerRef})`);
+        await page.evaluate(`render({useInnerRef: ${useInnerRef}})`);
+        await page.waitForSelector('#editableDiv');
+        const editComponent = f => page.evaluate('editComponent.' + f);
+        await testFun(page, editComponent);
+      });
+    }
+  }, 16000);
+}
+
+const MyComponent = () => React.createElement("div", {className: "MyComponent"});
+
+for (let tagName of [`"div"`, `"section"`, MyComponent]) {
```

Comment:
It would be better to avoid code duplication here. We could have a single loop that would go through
`{useInnerRef: true, tagName: 'div'}`, ``{useInnerRef: false, tagName: 'div'}``, `{useInnerRef: true, tagName: MyComponent}`, and so on...